### ISSUE TITLE
about training the cuhk03 dataset

### ISSUE THREAD

**Issue by rmxhhh** [layumi/2016_person_re-ID#22]
Hi, I am very sorry to bother you again.I change the line from `fc751Block = dagnn.Conv('size',[1 1 2048 751],'hasBias',true,'stride',[1,1],'pad',[0,0,0,0]);
` to `fc751Block = dagnn.Conv('size',[1 1 2048 1367],'hasBias',true,'stride',[1,1],'pad',[0,0,0,0]);
`,And no other changes have been made elsewhere.Then i run 'resnet52_2stream.m' to recreat 'net.mat', after that i run the 'train_id_net_res_2stream.m', but there has an error shows
`错误使用 gpuArray/subsref
Index exceeds matrix dimension.
出错 vl_nnloss (line 234)
      t = Xmax + log(sum(ex,3)) - x(ci) ;
出错 dagnn.Loss/forward (line 14)
      outputs{1} = vl_nnloss(inputs{1}, inputs{2}, [], 'loss', obj.loss, obj.opts{:}) ;
出错 dagnn.Layer/forwardAdvanced (line 85)
      outputs = obj.forward(inputs, {net.params(par).value}) ;
出错 dagnn.DagNN/eval (line 91)
  obj.layers(l).block.forwardAdvanced(obj.layers(l)) ;
出错 cnn_train_dag>processEpoch (line 222)
      net.eval(inputs, params.derOutputs, 'holdOn', s < params.numSubBatches) ;
出错 cnn_train_dag (line 90)
    [net, state] = processEpoch(net, state, params, 'train',opts) ;
出错 train_id_net_res_2stream (line 34)
[net,info] = cnn_train_dag(net, imdb, @getBatch,opts) ;`
Can you tell me what's wrong with me?Thank you very much!

**Comment by rmxhhh** [comment]
I solved it myself.I download the  new ZIP from GITHUB again and run the cuhk03 dataset. And it works! Maybe it's because I changed a lot of lines when I used Market1501 dataset.
